### PR TITLE
add Samuel to release testing rotation

### PR DESCRIPTION
**Pull request by garyverhaegen-da**

### CODE REVIEW

**Review by basvangijzel-DA — APPROVED** [review]
Thanks Gary!

**Comment by samuel-williams-da** [comment]
So it begins :p

**Comment by garyverhaegen-da** [comment]
> So it begins :p

You mentioned your interest in automating those, so I thought I'd give you some incentive.

File: release/rotation [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # If you are adding someone to the rotation, remember to also give them permission
 # to access an ad-hoc machine for Windows testing. This can be done by opening a PR
 # like the following: https://github.com/DACH-NY/daml-language-ad-hoc/pull/37
+U04M62HVADS samuel-williams-da
```
Comment (nickchapman-da) [inline release/rotation:4]: Added at the top. Nice! :)

**Pull request merged**